Ctor 1045 monitoring connectors add protocol in sidebar titles when missing (#4859)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-3com-snmp.md

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias        | Service Template                          | Service Description                          | Discovery |
|:---------------------|:------------------------------------------|:---------------------------------------------|:---------:|

> | Traffic-Generic-Id   | Net-3com-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface        |           |

Reviewer: ```suggestion
| Traffic-Generic-Id   | Net-3com-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface        |           |
```

> | Traffic-Generic-Id   | Net-3com-Traffic-Generic-Id-SNMP-custom   | Check traffic of an network interface        |           |

Reviewer: ```suggestion
| Traffic-Generic-Id   | Net-3com-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface        |           |
```

> | Traffic-Generic-Name | Net-3com-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface        |           |

Reviewer: ```suggestion
| Traffic-Generic-Name | Net-3com-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface        |           |
```

> | Traffic-Generic-Name | Net-3com-Traffic-Generic-Name-SNMP-custom | Check traffic of an network interface        |           |

Author: ```suggestion
| Traffic-Generic-Id   | Net-3com-Traffic-Generic-Id-SNMP-custom   | Check traffic of a network interface        |           |
| Traffic-Generic-Name | Net-3com-Traffic-Generic-Name-SNMP-custom | Check traffic of a network interface        |           |
```
